securedrop/Makefile

```diff
@@ -1,79 +1,71 @@
 DEFAULT_GOAL: help
-TAG ?= $(shell git rev-parse HEAD)
-IMAGE = securedrop
 
 .PHONY: lint
-lint: ## Run the python linter with the project's default error settings
-	find . -name '*.py' | xargs pylint --reports=no --errors-only \
-	   --disable=no-name-in-module \
-	   --disable=unexpected-keyword-arg \
-	   --disable=too-many-function-args \
-	   --disable=import-error \
-	   --disable=no-member \
-	   --max-line-length=100
+lint: ## Run all linters
+	@echo "\n    ►►► securedrop/Makefile is deprecated. Run 'make lint' in the repo root. ◀◀◀\n"
```

Comment:
let's file a followup after this PR is merged to update dev docs (since things Still Work here not going to append here since this diff is already large), but still worth doing next so that the Happy Contributor Path doesn't involve hitting deprecation warnings

Reply:
OK.